spec/controllers/users_controller_spec.rb

```diff
@@ -68,15 +77,31 @@
     end
   end
 
-  describe 'DELETE users#destroy' do
+  describe 'users#destroy' do
     it 'deletes the user' do
       expect(response).to have_http_status(:ok)
       expect { delete :destroy, params: { id: user.id } }.to change(User, :count).by(-1)
     end
 
     it 'does not delete any user if the user id is invalid' do
```

Comment:
I think that this should be scoped inside of a test group like `context 'current_user has NOT 'ManageUsers' permission'`,
And maybe we can add two tests one for `it 'returns status code :forbidden for a NONE current_user :id '` targeting an existing user and another `it 'returns status code :forbidden for inexistent :id '` targeting an inexistent user,